README.md

```diff
@@ -22,6 +22,7 @@ A curated list of awesome things and projects built by Algerian developpers.
 - [Salat Time](https://github.com/Barbarosse/salat_time/) A simple python Web App that returns the five Islamic Prayer Times.
 - [selma](https://github.com/FdevTech/Selma) - A virtual assistant backed by augmented reality capabilities, it allows you to control different appliances in your home.
 - [Timgad](https://github.com/OpenDZ/timgad) - Timgad is a kernel Linux Security Module that collects per process and system-wide security protections that are not handled by the core kernel itself.
+- [LSQLApp](https://github.com/houarizegai/lsqlapp) - Desktop application that can help teacher in practical work (TP) to evaluate the student, and help student to learn SQL (structure query language), "Learning by doing".
```

Comment:
```suggestion
- [LSQLApp](https://github.com/houarizegai/lsqlapp) - Desktop application that helps teachers in practical work (TP) to evaluate students. It also helps students to learn SQL.
```

Here's a small suggestion to clarify the wording. Can you move it up to keep the alphabetical order of the list?  Thanks.